tests/test_encoder.py

```diff
@@ -213,3 +213,20 @@ def test_encode_property_list(self):
             expected_geometry=[[0, 0]],
             properties=properties,
             expected_properties={'test_empty': ''})
+
+    def test_encode_multiple_values(self):
+        geometry = 'POINT(0 0)'
+        properties = dict(foo='bar')
```

Comment:
Yea, I did it that way originally, but then I remembered that we had problems in the past with the ordering of the feature properties during iteration when testing binaries. I expect that the multiple features would also trigger this scenario without any of the consequences.

Or do you think that it isn't a valid test for this case?